lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -235,22 +261,27 @@ def bulk_create_unsaved(self):
         """
         Bulk creates all the unsaved subsection_grades to this point.
         """
+        self._log_event(log.warning, u"bulk_create_unsaved")
+
         SubsectionGrade.bulk_create_models(self.student, self._unsaved_subsection_grades, self.course.id)
         self._unsaved_subsection_grades = []
 
     def update(self, subsection, block_structure=None):
         """
-        Updates the SubsectionGrade object for the student and subsection
-        identified by the given usage key.
+        Updates the SubsectionGrade object for the student and subsection.
```

Comment:
Are we still anti-one line docstring? I don't have a problem with them personally.